main.js

```diff
@@ -0,0 +1,71 @@
+var resetTimer = function(){
+  clearInterval(secTimerClock);
+  timerMin.innerHTML = '25';
+  timerSec.innerHTML = ':00';
+  minNum = 24;
+  secNum = 60;
+  taskInput.remove();
+  receiveInput();
+}
+
+var pauseTimer = function(){
+  clearInterval(secTimerClock);
+  startbtn.removeEventListener('click',function(){});
+  startbtn.addEventListener('click', startSecTimer);
+
+}
+
+var secCountDown = function(){
+  timerMin.textContent = minNum;
+  timerSec.textContent = ':' + secNum;
+  secNum -= 1;
+  if (secNum < 10){
+    timerSec.textContent = ':0' + secNum;
+  }else {
+    timerSec.textContent = ':' + secNum;
+  }
+
+  if (secNum === 0 && minNum === 0){
+    clearInterval(secTimerClock);
+    window.setTimeout(function(){
+    alert("Great Job!!! It's time to take a break!!!");
+    },500);
+  } else if (secNum === 0){
+    secNum = 60;
+    minNum -= 1;
```

Comment:
Interesting way to work through minutes and seconds. You could also rely on 25 * 60 * 1000 milliseconds to countdown from and then format that string.